Makefile

```diff
@@ -247,6 +247,40 @@ verify-crd:
 	$(diff) '$(tmp_file)' '$(CRD_PATH)' || (echo 'File $(CRD_PATH) is not up-to date. Please run `make update-crd` to update it.' && false)
 .PHONY: verify-crd
 
+# $1 - target file
+define generate-helm-schema-scylla
+	$(YQ) eval -j '{"$$schema": "http://json-schema.org/schema#"} * (.spec.versions[] | select(.name == "v1") | \
+	 .schema.openAPIV3Schema.properties.spec) | \
+	 .properties.racks=.properties.datacenter.properties.racks | \
+	 .properties.datacenter={"type": "string"}' $(CRD_PATH) > '$(1)'
+endef
+
+# $1 - Scylla schema
+# $2 - target file
+define generate-helm-schema-scylla-manager
+	$(YQ) eval-all -j 'select(fi==0).properties.scylla = (select(fi==1) | \
+	del(.$$schema)) | \
+	select(fi==0)' $(HELM_CHARTS_DIR)/scylla-manager/template.values.schema.json '$(1)' > '$(2)'
+endef
+
+update-helm-schemas:
+	$(call generate-helm-schema-scylla,'$(HELM_CHARTS_DIR)/scylla/values.schema.json')
+	$(call generate-helm-schema-scylla-manager,'$(HELM_CHARTS_DIR)/scylla/values.schema.json','$(HELM_CHARTS_DIR)/scylla-manager/values.schema.json')
+.PHONY: update-helm-schemas
+
+verify-helm-schemas: tmp_dir:=$(shell mktemp -d)
+verify-helm-schemas:
+	mkdir -p $(tmp_dir)/{scylla,scylla-manager}
+
+	cp '$(HELM_CHARTS_DIR)/scylla/values.schema.json' '$(tmp_dir)/scylla/values.schema.json'
```

Comment:
why do we need a pre-copy of the file we are writing into?

Reply:
leftovers from previous version that went unnoticed, sorry